cloud/template.yml

```diff
@@ -31,3 +31,58 @@ Resources:
                   - 'dynamodb:Query'
                 Resource: '*'
   # End IAM Roles
+
+  userTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      AttributeDefinitions:
+        - AttributeName: 'id'
+          AttributeType: 'S'
+        - AttributeName: 'faculty'
+          AttributeType: 'S'
+      KeySchema:
+        - AttributeName: 'id'
+          KeyType: 'HASH'
+      ProvisionedThroughput:
```

Comment:
When you set the ProvisionedThoroughput, it means that the billing mode is 'Provisioned'. As you can see by [this](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BillingModeSummary.html) doc, you can see that Provisioned is great for predictable workloads but PAY_PER_REQUEST is great for unpredictable workloads (and only charges per request). For all our tables and indexes, I think this fits better. 

What do you think?

Reply:
Yep having a read of some different resources it does look like Pay PerRequest looks like the way to go.